gcc/rust/expand/rust-derive-clone.cc

```diff
@@ -144,8 +144,29 @@ DeriveClone::visit_enum (Enum &item)
 void
 DeriveClone::visit_union (Union &item)
 {
-  rust_sorry_at (item.get_locus (), "cannot derive %qs for these items yet",
-		 "Clone");
+  // FIXME: Should be $crate::core::clone::AssertParamIsCopy (or similar)
```

Comment:
Raise an issue about the whole path to the lang item stuff because there is overlap with what I am doing there too

Reply:
Done in #2263